<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSerieDialog</class>
 <widget class="QDialog" name="AddSerieDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="descriptionTextEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="genreLabel">
       <property name="text">
        <string>Genre</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QListWidget" name="genreList"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QListWidget" name="statusList"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="productionuYearLabel">
       <property name="text">
        <string>Production year</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="productionYearSpinBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="seasonCountLabel">
       <property name="text">
        <string>Season count</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="seasonCountSpinBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="creatorLabel">
       <property name="text">
        <string>Creator</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QListWidget" name="creatorList"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="markLabel">
       <property name="text">
        <string>Mark</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="watchedLabel">
       <property name="text">
        <string>Watched</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="watchedCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="sliderLayout">
       <item>
        <widget class="QSlider" name="markSlider">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="markValueLabel">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddSerieDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddSerieDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
